### ISSUE TITLE
Find in files in non-existent folder highlights wrong input field on error

### ISSUE THREAD

**Issue by mgabeler-lee-6rs** [Microsoft/vscode#67068]
Issue Type: <b>Bug</b>

If I mistakenly enter a folder that does not exist in the "files to include" box for Find in Files and hit enter to search, the red error highlight ends up on the search expression input, not on the files to include input.  As a side effect, the actual error message ("No folder in the workspace with name: ...") only shows up when the input focus is on the search expression field, not the files to include field that has the error.

Verified this still happens with all extensions disabled.

Example:
![image](https://user-images.githubusercontent.com/41014492/51690837-89102380-1fc7-11e9-971c-7e4a295671d6.png)

VS Code version: Code 1.30.2 (61122f88f0bf01e2ac16bdb9e1bc4571755f5bd8, 2019-01-07T22:49:48.319Z)
OS version: Linux x64 4.19.0-1-amd64

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz (8 x 2809)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: disabled_software<br>video_decode: unavailable_software<br>video_encode: unavailable_software<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|4, 4, 3|
|Memory (System)|15.55GB (0.15GB free)|
|Process Argv|--unity-launch|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (24)</summary>

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|10.2.0
ng-template|Ang|0.1.11
markdown-preview-github-styles|bie|0.1.4
bracket-pair-colorizer-2|Coe|0.0.25
vscode-markdownlint|Dav|0.23.0
vscode-eslint|dba|1.8.0
gitlens|eam|9.4.1
EditorConfig|Edi|0.12.6
tslint|eg2|1.0.42
vscode-npm-script|eg2|0.3.5
prettier-vscode|esb|1.7.3
vscode-pull-request-github|Git|0.3.2
beautify|Hoo|1.4.7
bash-ide-vscode|mad|1.3.3
terraform|mau|1.3.7
Go|ms-|0.8.0
debugger-for-chrome|msj|4.11.1
vscode-docker|Pet|0.5.1
bash-debug|rog|0.3.3
vscode-coverage-gutters|rya|2.3.0
vscode-nginx|sha|0.6.0
html-preview-vscode|tht|0.2.5
reflow-paragraph|Tro|1.3.0
vscode-proto3|zxh|0.2.2

(1 theme extensions excluded)

</details>

**Comment by andreamah** [comment]
I don't think that I can reproduce this bug, are you still seeing it?

**Comment by mgabeler-lee-6rs** [comment]
Yes, it still reproduces the same as my original report. You have to hit enter in the find field to actually try to run the search, not just type the values

**Comment by andreamah** [comment]
Yes, I performed the search. You're still getting the red border?

[![Image from Gyazo](https://i.gyazo.com/6197788bb9ed1acd0ea7af8cee35e483.gif)](https://gyazo.com/6197788bb9ed1acd0ea7af8cee35e483)

**Comment by mgabeler-lee-6rs** [comment]
Perhaps it's related to having a multi-root workspace
![Peek 2022-12-15 14-18](https://user-images.githubusercontent.com/41014492/207948120-48f6093e-bc10-4be7-b6fa-9a3b82a0cf76.gif)

**Comment by andreamah** [comment]
Oh I was able to see it 
<img width="266" alt="image" src="https://user-images.githubusercontent.com/31675041/208465295-b53df34d-651a-4fdf-b42a-63d3c1b1d588.png">
